File: android/ifood/app/src/main/java/br/com/mayconsantana/ifood/activity/MainActivity.java
package br.com.mayconsantana.ifood.activity;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

import br.com.mayconsantana.ifood.R;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        getSupportActionBar().hide();
    }
}

File: android/Sorteiokotlin/app/src/main/java/com/mayconsantana/sorteio_kotlin/MainActivity.kt
package com.mayconsantana.sorteio_kotlin

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.TextView
import java.util.*

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun sortearNumero(view: View) {

        var texto = findViewById<TextView>(R.id.testoSorteio)

        var numeroSorteado = Random().nextInt(11)

        texto.setText("Número sorteado é: " + numeroSorteado)

    }

}